snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedRequests.java

```diff
@@ -468,6 +468,10 @@ public static SnomedConceptSearchRequestBuilder prepareGetSynonyms() {
 					constraintBulkRequestBuilder.add(SnomedRequests.prepareSearchConstraint().all().filterBySelfIds(selfIds));
 				}
 				
+				if (!CompareUtils.isEmpty(ruleParentIds)) {
```

Comment:
⚠️ Please check the callers of this method: `OpenSnomedConceptInEditor` takes all direct and indirect parents, both stated and inferred, and passes it via this parameter.